ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml

```diff
@@ -229,6 +229,24 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi
+- as: e2e-metal-ovn-sno-cert-rotation-shutdown
+  optional: true
```

Comment:
Lets add `always_run: false` so that these would not start on every PR but instead can be started ad-hoc?

Reply:
we could, @hasbro17 wdyt?